prisma/schema.prisma

```diff
@@ -10,6 +10,27 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model reactboard {
+	id Int @id @default(autoincrement())
+	guildId String
+	channelId String
+	react String
+	isCustomReact Boolean
+	threshold Int
+
+	@@unique([guildId, channelId], name: "location")
+
+	reactboardPosts reactboardPost[]
+}
+
+model reactboardPost {
+	id Int @id @default(autoincrement())
+	reactboardId Int
+	reactboard reactboard @relation(fields: [reactboardId], references: [id])
+	originalMessageId String
```

Comment:
As well as an `originalMessageId`, I would like if we also include an `originalChannelId`, since Discord has no firm guarantee that a message ID is unique outside of its channel.